scripts/o3de/o3de/export_project.py

```diff
@@ -1092,6 +1089,14 @@ def register_setting(key: str, description: str, default: str = None, is_passwor
                                                             is_boolean=True,
                                                             default='False')
 
+SETTINGS_DEFAULT_BUILD_TOOLS_PATH = register_setting(key='default.build.tools.path',
+                                                     description='Designates where the build files for the O3DE toolchain are generated.',
+                                                     default='build/tools')
+
+SETTINGS_DEFAULT_LAUNCHER_TOOLS_PATH = register_setting(key='default.launcher.build.path',
+                                                        description='Designates where the launcher build files (Game/Server/Unified) are generated.',
+                                                        default='build/launcher')
```

Comment:
fwiw I'm not a fan of these default build paths not matching anything we have in our CMakePresets.json,  Would be nice if it output to what the CMakePresets would use, but that can be a separate PR.

Reply:
Yea i agree it should be a separate PR. Fwiw it at least doesn't affect the immediate scope of script correctness. I'll make a GHI after this PR's merged

Reply:
We don't use the CMakePresets in this use case though. Maybe we should? But then we would also have to define presets for the export use case as well, and we can also get the cmake configurations from the presets as well rather than needing to hard code them.

Reply:
the downside of the user getting this wrong is very costly.  It is so easy to forget to pass in your existing build tools path e.g. `build/window` or `build/windows_vs2022` and have this export command take an additional hour to rebuild the tools in a different path rather than using the existing built tools.

Reply:
Lets make it a separate GHI/PR for the future